examples/che_devfile.yaml

```diff
@@ -0,0 +1,31 @@
+specVersion: 0.0.1
```

Comment:
For this examples/che_devfile.yaml file, do you have code that can generate it once it needs to? Like a golden file?

Reply:
@kwk do you mean `component_cr.yml` file which would generate this `che_devfile` ?